### DISCUSSION TITLE
Support for styled-component

### DISCUSSION THREAD

**Discussion by subhendukundu** [flareact/flareact#113] [Q&A] [answered]
I am keeping a track of flareact for a while now. I love the fact that its growing really fast and it should.

A quick question, are we supporting styled-components or similar yet? If not would love to contribute where I can.
Thanks

**Comment by traffisco** [comment, marked as answer]
> I am keeping a track of flareact for a while now. I love the fact that its growing really fast and it should.
> 
> A quick question, are we supporting styled-components or similar yet? If not would love to contribute where I can.
> Thanks

https://github.com/flareact/flareact/tree/canary/examples/with-styled-components

**Reply by subhendukundu** [reply]
Hey! Sorry took me a while. So wanted to create few examples with popular frameworks react community likes to use.
Turned out flareact is not working with emotion. Getting a 500 error. I have pushed it in a repo. Any idea why?
https://github.com/subhendukundu/with-emotion

**Reply by subhendukundu** [reply]
Dang it, I guess I know the issue. We are in SSR. Hold on.

**Reply by unsphere** [reply]
> Hey! Sorry took me a while. So wanted to create few examples with popular frameworks react community likes to use.
> Turned out flareact is not working with emotion. Getting a 500 error. I have pushed it in a repo. Any idea why?
> https://github.com/subhendukundu/with-emotion

Hey, take a look at my discussion [here](https://github.com/flareact/flareact/discussions/136). I try to work with emotion (Chakra UI) as well and edited the _document.js file for styled-components, styled-jsx, and emotion.

**Reply by subhendukundu** [reply]
So it is an issue with emotion I guess. I added a comment or issue somewhere, I am not able to find it. Let me see if I can pull it from somehwhere.

**Reply by unsphere** [reply]
Okay, but SSR is supported by Emotion since v10. So normally we don't have to include it in the _document.js file.
